Incompatible with the latest Flutter beta (2.5.0-5.1.pre)

```
../../../../.pub-cache/hosted/pub.dartlang.org/assorted_layout_widgets-5.0.2/lib/src/text_one_line_ellipsis_with_fade.dart:995:20: Error: The method 'RenderParagraphX.getBoxesForSelection' has fewer named arguments than those of overridden method 'RenderParagraph.getBoxesForSelection'.
  List<ui.TextBox> getBoxesForSelection(TextSelection selection) {
```

Any update on this? its causing error with flutter 2.5 stable. Please fix this ASAP.

`/C:/SDKs/flutter/.pub-cache/hosted/pub.dartlang.org/assorted_layout_widgets-5.1.1/lib/src/text_one_line_ellipsis_with_fade.dart:995:20: Error: The method 'RenderParagraphX.getBoxesForSelection' has fewer named arguments than those of overridden method 'RenderParagraph.getBoxesForSelection'.
  List<ui.TextBox> getBoxesForSelection(TextSelection selection) {
                   ^
/C:/SDKs/flutter/packages/flutter/lib/src/rendering/paragraph.dart:811:20: Context: This is the overridden method ('getBoxesForSelection').
  List<ui.TextBox> getBoxesForSelection(
                   ^


FAILURE: Build failed with an exception.
`

--- a/lib/src/text_one_line_ellipsis_with_fade.dart
+++ b/lib/src/text_one_line_ellipsis_with_fade.dart
@@ -1,5 +1,6 @@
 import 'dart:math' as math;
-import 'dart:ui' as ui show Gradient, Shader, TextBox, PlaceholderAlignment;
+import 'dart:ui' as ui show Gradient, Shader, TextBox, PlaceholderAlignment, TextHeightBehavior, BoxHeightStyle, BoxWidthStyle;
+
 import 'package:flutter/foundation.dart';
 import 'package:flutter/gestures.dart';
 import 'package:flutter/material.dart';
@@ -992,7 +993,11 @@ class RenderParagraphX extends RenderBox
   ///
   /// Valid only after [layout].
   @override
-  List<ui.TextBox> getBoxesForSelection(TextSelection selection) {
+  List<ui.TextBox> getBoxesForSelection(
+      TextSelection selection, {
+        ui.BoxHeightStyle boxHeightStyle = ui.BoxHeightStyle.tight,
+        ui.BoxWidthStyle boxWidthStyle = ui.BoxWidthStyle.tight,
+      }) {
     assert(!debugNeedsLayout);
     _layoutTextWithConstraints(constraints);
     return _textPainter.getBoxesForSelection(selection);

